src/groundlight/client.py

```diff
@@ -408,35 +408,38 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments, t
         :param detector: the Detector object, or string id of a detector like `det_12345`
 
         :param image: The image, in several possible formats:
+
```

Comment:
The empty line is necessary in reStructuredText (otherwise the previous line is bolded in the sphinx docs)